jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -293,12 +311,20 @@ protected int getNumRetries() {
   @VisibleForTesting
   protected HiveConnection(String uri, Properties info,
       IJdbcBrowserClientFactory browserClientFactory) throws SQLException {
+    this(uri, info, browserClientFactory, true);
+  }
+
+  @VisibleForTesting
```

Comment:
why do we need this annotation, I don't see use in tests for the new constructor